### DISCUSSION TITLE
MSSQL docker compose is not working

### DISCUSSION THREAD

**Discussion by MathavanN** [keycloak/keycloak#13388] [Q&A]
I'm getting the below error when I start keycloak with MSSQL database
![image](https://user-images.githubusercontent.com/5356325/181467605-17d804e4-3794-4f70-bdd6-b9a8a9788f22.png)

**Comment by M2xAy** [comment]
> I'm getting the below error when I start keycloak with MSSQL database
> 
> ![image](https://user-images.githubusercontent.com/5356325/181467605-17d804e4-3794-4f70-bdd6-b9a8a9788f22.png)
> 
> 

Thats right. With mssql i have also same problem but only with latest version or up version 16. with version 16 works fine. Any ideas?
Best regards
